demo_scripts/Classes/custom_menu_bar_class.py

```diff
@@ -173,6 +175,77 @@ def _add_commands_and_buttons_open_files_folders(open_folders_menu_id, open_scri
                     if curr_file.rsplit(".", 1)[1] == "py" and curr_file != "__init__.py":
                         add_command("open_script_" + curr_file, lambda local_var=folder_path + "\\" + curr_file: (subprocess.Popen(["notepad.exe", local_var], shell=True)))
                         add_menu_item(open_scripts_menu_id, curr_file, "open_script_" + curr_file)
+
+    @staticmethod
+    def _add_help_root_menu_items(menu_id):
+        for curr_command in qtm.gui.get_commands():
+            if "help_root_" in curr_command:
```

Comment:
suggestion: use a constant for "help_root_"

Reply:
Renamed to "root_help_" and also made it a constant. I also moved the "leaking" menu-item creation code that was previously placed in the 'startup' file to the 'custom_menu_bar', which is a more correct location.